tests/RobotFramework/tests/tedge/diag/predefined_plugins.robot

```diff
@@ -20,10 +20,16 @@ Test Tags           theme:troubleshooting    theme:cli    theme:plugins
     FOR    ${log_name}    IN    @{log_names}
         File Size Is Not Zero    ${log_name}
     END
+
+    Log Should Contain    tedge-agent.log    Starting tedge-agent.service
```

Comment:
The earlier `File Size Is Not Zero` check is redundant now, as we're checking the content itself here.